### DISCUSSION TITLE
Incorrect background color being applied when using theme.colors

### DISCUSSION THREAD

**Discussion by amerfarooq** [mantinedev/mantine#3891] [Q&A] [answered]
In my theme, I have defined the following dark colors:

```ts
dark: [
  '#F8F8F8', // 0
  '#F3F4F6', // 1
  '#D4D4D8', // 2
  '#8E8EA4', // 3
  '#F3FAF7', // 4
  '#71717A', // 5
  '#52525B', // 6
  '#3F3F46', // 7
  '#27272A', // 8
  '#18181B', // 9
],
```

I am using an AppShell and setting a background color like so: 

```ts
ProjectOnboarding.getLayout = function getLayout(page: ReactElement) {
  return (
    <AppShell
      sx={(theme) => ({
        backgroundColor: '#F8F8F8',
      })}
    >
      {page}
    </AppShell>
  );
};
```

This sets the color correctly and I've verified this by using a color picker tool. However, if I set the same color from the theme instead, it does not apply properly i.e.

```ts
ProjectOnboarding.getLayout = function getLayout(page: ReactElement) {
  return (
    <AppShell
      styles={(theme) => ({
        main: {
          backgroundColor: theme.colors.dark[0],
        },
      })}
    >
      {page}
    </AppShell>
  );
};
```

Doing this ends up setting the color to `#C1C2C5` and I have no clue how this comes about. Can anyone point out what I'm doing wrong?

**Comment by Sajarin-M** [comment, marked as answer]
`#C1C2C5` is the default value for `theme.colors.dark[0]`. Are you sure you have successfully overridden the theme?

**Reply by amerfarooq** [reply]
Thanks for pointing that out. The Next.js layout wasn't wrapping the `MantineProvider` correctly so I guess it was using the default theme.
